src/store/modules/user/actions.ts

```diff
@@ -58,6 +58,9 @@ const actions: ActionTree<UserState, RootState> = {
       if (resp.data.userTimeZone !== localTimeZone) {
         emitter.emit('timeZoneDifferent', { profileTimeZone: resp.data.userTimeZone, localTimeZone});
       }
+      await UserService.getShippingOrderPreference({
```

Comment:
We are fetching the preference value but not using it to set the shipping order status.

Reply:
Fixing it sir!